crates/sui-types/src/object.rs

```diff
@@ -592,7 +592,9 @@ impl Object {
             previous_transaction,
         );
 
+        #[cfg(not(msim))]
```

Comment:
Made this change so that I could implement a test in msim mode that adds a new system package  -- I can also introduce a dummy. ID for a system package in `msim` mode if that's preferable.